apps/mobile/src/features/account-selector-sheet.tsx

```diff
@@ -60,7 +61,7 @@ export function AccountSelectorSheet({ sheetRef }: { sheetRef: RefObject<SheetRe
   );
 
   return (
-    <Sheet isScrollView shouldHaveContainer={false} ref={sheetRef} themeVariant={themeVariant}>
+    <Sheet isScrollView ref={sheetRef} themeVariant={themeVariant}>
```

Comment:
Did you mean to remove `shouldHaveContainer`? This seems to be the only place that uses it to avoid wrapping sheet children with `BottomSheetComponent`.

Reply:
Currently, the My Accounts button is broken

<img width="552" alt="image" src="https://github.com/user-attachments/assets/b9eb20ed-9903-4897-89ff-1377dfd12230">

Removing this seems to fix it